src/main/java/nl/workingtalent/bookrental/controller/UserController.java

```diff
@@ -26,24 +27,26 @@ public class UserController {
 	@Autowired
 	private IUserRepository repo;
 	
+	@Autowired
+	private EmailService emailService;
+	
 	@PostMapping("user/create")
 	public String createUser(@RequestHeader(name = "Authorization") String token, @RequestBody NewUserDto newUserDto) {
+		
 		User loggedInUser = repo.findByToken(token);
 
+		// Check if user had Admin rights
 		if (loggedInUser == null || !loggedInUser.isAdmin()) {
 			return "No permission";
 		}
 		
 		// Check if user already exists in database by email
 		User existingUser = repo.findByEmail(newUserDto.getEmail());
 		if (existingUser != null) {
-			System.out.println("Email already exists.");
-			return "Email exists";
+			return "Email already exists";
 		}
-
+		
 		String encodedPassword = passwordEncoder.encode(newUserDto.getPassword());
-		System.out.println("The default ecoded password is " + encodedPassword);
-
 		User user = new User();	
```

Comment:
De firstname en lastname willen we eigenlijk in de constructor meegeven -> 
User user = new User(newUserDto.getFirstName(), newUserDto.getLastName());